### PR TITLE
open-scene-graph: fix compatibility with collada-dom 2.4

### PR DESCRIPTION
**Pull request by jcarpent**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by jcarpent** [comment]
@ilovezfs I really do understand what is wrong with this PR. Maybe, you can help me on this bug?
The idea is to make this Formula compatible with the installation of the osg-dae plug-in.

**Comment by ilovezfs** [comment]
try `ENV["SDKROOT"] = MacOS.sdk_path`

**Comment by jcarpent** [comment]
@ilovezfs Thanks for your quick reply.

**Comment by jcarpent** [comment]
It perfectly works.

File: Formula/open-scene-graph.rb [hunk 1]
```diff
@@ -69,7 +69,11 @@ def install
     end
 
     if build.with? "collada-dom"
-      args << "-DCOLLADA_INCLUDE_DIR=#{Formula["collada-dom"].opt_include}/collada-dom"
+      if Formula["collada-dom"].version >= "2.4"
```
Comment (fxcoudert) [inline Formula/open-scene-graph.rb]: No conditional, we don't support older versions.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Comment by jcarpent** [comment]
@fxcoudert I fixed the requested changes.

**Comment by ilovezfs** [comment]
Thanks for your first contribution to Homebrew, @jcarpent! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**